e2e/playwright/mock_oauth_server/mock_oauth_server.ts

```diff
@@ -0,0 +1,55 @@
+import express from 'express';
+
+export enum ExpiryAlgorithms {
+    NO_EXPIRY = 'no_expiry',
+}
+
+export type OAuthServerOptions = {
+    mattermostSiteURL: string;
+    pluginId: string;
+    mockOAuthAccessToken: string;
+    authorizeURLPrefix: string;
+    expiryAlgorithm: ExpiryAlgorithms;
+}
+
+export const makeOAuthServer = ({
+    mattermostSiteURL,
+    pluginId,
+    mockOAuthAccessToken,
+    authorizeURLPrefix,
+    expiryAlgorithm,
+}: OAuthServerOptions): express.Express => {
+    console.log(`Using Mattermost Site URL: ${mattermostSiteURL}`);
+    console.log(`Running OAuth server for plugin: ${pluginId}`);
```

Comment:
can we remove console.log? they appear in the output once + each time a test fails